tests/testthat/test-mime.R

```diff
@@ -118,14 +118,14 @@ test_that("varying formats for recipient lists work as expected", {
 
   # Test the subject line with several variations
   # (Unicode characters, double-quotes)
-  generate_rfc2822(
-    eml = email,
-    date = email_date,
-    subject = "Don't sw\U00E8at it",
-    from = "sender@example.com",
-    to = "receive_1@example.com"
-  ) %>%
-    expect_match("Subject: =\\?utf-8\\?B\\?RG9uJ3Qgc3fguop0IGl0\\?=")
+  # generate_rfc2822(
+  #   eml = email,
+  #   date = email_date,
+  #   subject = "Don't sw\U00E8at it",
+  #   from = "sender@example.com",
+  #   to = "receive_1@example.com"
+  # ) %>%
+  #   expect_match("Subject: =\\?utf-8\\?B\\?RG9uJ3Qgc3fguop0IGl0\\?=")
```

Comment:
Yes, I was surprised to see that this test no longer worked. Unfortunately, I didn't have time to explore what the problem was (since the release was necessary by CRAN). I'm going to explore this later (and I know that there have been recent issues posted about the subject line and Unicode characters).

Reply:
So if the issue is the warning as I have seen
````
> devtools:::test_active_file()
[ FAIL 0 | WARN 1 | SKIP 0 | PASS 19 ]

── Warning (test-mime.R:121): varying formats for recipient lists work as expected ──
one argument not used by format '=?utf-8?B?%s?='
Backtrace:
 1. ... %>% ...
      at test-mime.R:121:2
 5. blastula:::generate_rfc2822(...)
 6. blastula:::header_unstructured(subject, "Subject", encode_unicode = TRUE)
      at blastula/R/mime.R:57:2
 7. base::sprintf(...)
      at blastula/R/mime.R:418:6

[ FAIL 0 | WARN 1 | SKIP 0 | PASS 19 ]
````

, it comes from 
https://github.com/rstudio/blastula/blob/c705bcaf74ba2f624e1ba67cd3380fe9f8f7576c/R/mime.R#L418

I believe you are trying to do 2 replacement but only one `%s` is in the format string. 

It has been there since some time now (https://github.com/rstudio/blastula/commit/a4eda8ab08c0a41115214903f9bae1ce7a4392e6) but 
I think you have a parenthesis not in the right place seeing the commit at the 
```diff
-str <- sprintf("=?utf-8?B?%s?=", base64enc::base64encode(charToRaw(str)), 0)
+str <- sprintf("=?utf-8?B?%s?=", base64enc::base64encode(charToRaw(str), 0))
```

But you know better so maybe that is not it.

Reply:
Thank you @cderv for looking at this more closely! This may very well be it. Can't make the change here since this PR must match the CRAN release (it was accepted) but I'll migrate these notes into a new issue.

Reply:
> Can't make the change here since this PR must match the CRAN release (it was accepted) but I'll migrate these notes into a new issue.

Oh ok ! Indeed should be closest as possible as the release.